src/encoded/audit/file.py

```diff
@@ -349,7 +367,7 @@ def audit_duplicate_quality_metrics(value, system):
     'audit_paired_with': audit_paired_with,
     'audit_specifications': audit_file_format_specifications,
     'audit_controlled_by': audit_file_controlled_by,
-    'audit_duplicate_quality_metrics': audit_duplicate_quality_metrics,
+    'audit_duplicate_quality_metrics': audit_duplicate_quality_metrics
```

Comment:
I think PEP standard is to end with a ','?

Reply:
Fixed.